x-pack/plugins/ingest_manager/server/plugin.ts

```diff
@@ -273,6 +277,15 @@ export class IngestManagerPlugin
 
     return {
       esIndexPatternService: new ESIndexPatternSavedObjectService(),
+      packageService: {
+        getInstalledEsAssetReferences: async (
+          savedObjectsClient: SavedObjectsClientContract,
+          pkgName: string
+        ): Promise<EsAssetReference[]> => {
+          const installation = await getInstallation({ savedObjectsClient, pkgName });
+          return installation?.installed_es || [];
+        },
+      },
```

Comment:
@jfsiii please let me if you want this moved to another file.

Reply:
Thanks for asking. I think it'd ideally it'd be defined elsewhere, but I'm fine shipping as-is and leaving that for later. I'll find/create a ticket and reference this there.